packages/pagination/src/docs/Canvas.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+
+import { Pagination } from '../Pagination';
+import { PaginationItem } from '../PaginationItem';
+
+import Link from './Link';
+
+export const PaginationCanvas = () => {
+  return (
+    <div style={{ maxWidth: '700px', margin: '0 auto' }}>
+      <Pagination currentPage={1} totalPages={1} />
+      <Pagination currentPage={1} totalPages={2} />
+      <Pagination currentPage={1} totalPages={3} />
+      <Pagination currentPage={1} totalPages={4} />
+      <Pagination currentPage={2} totalPages={4} />
+      <Pagination currentPage={1} totalPages={5} />
+      <Pagination currentPage={2} totalPages={5} />
+      <Pagination currentPage={3} totalPages={5} />
+      <Pagination
+        aria-label={'laaaabel'}
```

Comment:
I know this is an example for everybody, but could you change it with "pagination-label", please? :)